modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/DocumentLibrary/DocumentLibrary.es.js

```diff
@@ -429,6 +452,32 @@ const Main = ({
 		return true;
 	};
 
+	const isObjectInvalidExtension = (value) => {
```

Comment:
@nicolas-sss Could you please rename `isObjectInvalidExtension` to `isObjectFieldInvalidExtension`.

Reply:
Done!